### DISCUSSION TITLE
Issues connecting to Web Interface

### DISCUSSION THREAD

**Discussion by HundleBun** [Security-Onion-Solutions/securityonion#6216] [Unsupported Versions] [answered]
Recently installed SO 2.3.80 ISO using Doug's instructional vids on Youtube. Set up with 2 NICs (first is NAT, second is Bridged with allow promiscuous). Ran so-status and everything is in OK status. so-allow has been run for 192.168.0.0/16 range and set connect via hostname on initial install. Edited /etc/hosts file (Windows 10) to include the IP address of the SO vm and hostname for resolution. Can ping my host computer but cannot ping the SO vm from host. Checked documentation on site as well and can't narrow down there.

host ip range 192.168.0.0/24
SO ip 10.0.2.15

When attempting to connect to the web interface I get a time out error. It seems I missed a step along the way or misconfigured something maybe? It seems like SO is not responding to pings/connections from my host at all.

I'm new to SO and attempting to setup a virtual lab environment for training/testing. Any assistance would be appreciated!

Edit: Forgot to mention that I also tried to access from a Win10 vm I spun up and can't reach it from there either.

**Comment by dougburks** [comment]
Seems like a basic networking issue. Double-check your settings for IP address, subnet mask, gateway, and DNS. Also try rebooting your Windows 10 host in case the virtual networks are in a strange state.

> also tried to access from a Win10 vm I spun up and can't reach it from there either.

Is the Win10 vm on the 10.0.2.x network? Did you so-allow that range?

**Reply by HundleBun** [reply]
The Win10 vm was on the same 10.0.2.x network and subnet and I did so-allow that range as well. I believe the issue is related to what tomjones suggested below in regards to having the adapter set up on NAT. In which case this would also be why I didn't have VM to VM communication. I plan to change to bridged and continue testing.

**Comment by tomjones1977** [comment, marked as answer]
It's almost certainly the NAT from your security onion to the host, which will allow outbound but not inbound traffic.  You'll either need to set it up as a bridge, or port forward into your seconion VM (how this is done depends on what virtualisation platform you are using).

**Reply by HundleBun** [reply]
Thanks Tom, I'm going to go ahead and change that over to a bridged connection so I'm not altering too many port forwards within the lab environment I'm trying to create.

**Reply by HundleBun** [reply]
I wasn't sure of all the correct commands to alter configurations to change after setting it as bridged so I just spun it back up from a snapshot and reconfigured from setup. Now that I have it set NIC1Bridged/NIC2Internal everything seems to be working as intended. Thanks for the help!
